README.md

```diff
@@ -222,6 +223,52 @@ In a bundle, variables can come from 4 sources. Those sources and their preceden
 That is to say, variables set using the `--set` flag take precedence over all other variable sources.
 
 
+## Duplicate Packages And Naming
+
+It is possible to deploy multiple instances of the same Zarf package in a bundle. For example, the following `uds-bundle.yaml` deploys 3 instances of the [helm-overrides](src/test/packages/helm/zarf.yaml) Zarf packags:
+```yaml
+kind: UDSBundle
+metadata:
+   name: duplicates
+   description: testing a bundle with duplicate packages in specified namespaces
+   version: 0.0.1
+
+packages:
+   - name: helm-overrides
+     repository: localhost:5000/helm-overrides
+     ref: 0.0.1
+     overrides:
+        podinfo-component:
+           unicorn-podinfo: # name of Helm chart
+              namespace: podinfo-ns
+
+   # note the unique name and namespace
+   - name: helm-overrides-duplicate
+     repository: localhost:5000/helm-overrides
+     ref: 0.0.1
+     overrides:
+        podinfo-component:
+           unicorn-podinfo:
+              namespace: another-podinfo-ns
+
+   # note the unique name, namespace and the path to the Zarf package tarball
+   - name: helm-overrides-local-duplicate
+     path: src/test/packages/helm/zarf-package-helm-overrides-arm64-0.0.1.tar.zst
+     ref: 0.0.1
+     overrides:
+        podinfo-component:
+           unicorn-podinfo:
+              namespace: yet-another-podinfo-ns
+```
+
+The naming conventions for deploying duplicate packages are as follows:
+1. The `name` field of the package in the `uds-bundle.yaml` must be unique
+1. The duplicate packages must be deployed in different namespaces
+1. In order to deploy duplicates of local packages, the `path` field must point to a Zarf package tarball instead of to a folder; this allows us to satisfy the first convention.
```

Comment:
```suggestion
1. In order to deploy duplicates of local packages, the `path` field must point to a Zarf package tarball instead of to a folder.
```

Reply:
(nitpick) thought the last section felt unnecessary and made it a little confusing.